api/urls.py

```diff
@@ -39,23 +41,23 @@
     r"non_teaching_help", NonTeachingHelpViewSet, basename="non_teaching_help"
 )
 
-public_router = DefaultRouter()
-public_router.register(r"students", PublicStudentViewSet, basename="students")
-public_router.register(
+dashboard_router = DefaultRouter()
+dashboard_router.register(r"students", DashboardStudentViewSet, basename="students")
+dashboard_router.register(
     r"students_with_personal_info",
-    PublicStudentWithPersonalInfoViewSet,
+    DashboardStudentWithPersonalInfoViewSet,
     basename="students_with_personal_info",
 )
-public_router.register(r"groups", PublicGroupViewSet, basename="groups")
-public_router.register(r"teachers", PublicTeacherViewSet, basename="teachers")
-public_router.register(
+dashboard_router.register(r"groups", DashboardGroupViewSet, basename="groups")
+dashboard_router.register(r"teachers", DashboardTeacherViewSet, basename="teachers")
+dashboard_router.register(
     r"teachers_with_personal_info",
-    PublicTeacherWithPersonalInfoViewSet,
+    DashboardTeacherWithPersonalInfoViewSet,
     basename="teachers_with_personal_info",
 )
 
-# "Internal" API is used by bot and "public" API is used by the Tooljet
+# "Internal" API is used by bot and "dashboard" API is used by the Tooljet
 urlpatterns = [
     path("", include(internal_router.urls)),
-    path("public/", include(public_router.urls)),
+    path("dashboard/", include(dashboard_router.urls)),
```

Comment:
We might want to leave the URL path name "public/" as it is if it would require too much effort from @ivanBarinsky to change URLs in the dashboard. Let's see what Ivan says :)
(Same for other cases)

Reply:
It wouldn't cause too many issues on my side, i think its a good time to do it now since im not using too many queries in tooljet quite yet 👍

Reply:
Happy to resolve this on my side

Reply:
> It wouldn't cause too many issues on my side, i think its a good time to do it now since im not using too many queries in tooljet quite yet +1

OK, let's do `dashboard/` then :)